src/index.js

```diff
@@ -223,12 +225,18 @@ class CoreConsoleAPI {
     } else {
       // init swagger client
       const spec = require('../spec/api.json')
-      const swagger = new Swagger({
+      const options = {
         spec,
         requestInterceptor: requestInterceptorBuilder(this, apiHost),
         responseInterceptor,
         usePromise: true
-      })
+      }
+      if (process.env.HTTPS_PROXY) {
+        logger.debug('Using proxy - ' + process.env.HTTPS_PROXY)
+        options.userFetch = proxyFetch
```

Comment:
do we want to always use our proxyFetch? i.e. always set `userFetch` to proxyFetch.
This way we don't have to do this if statement since proxyFetch will do this internally

Reply:
This check was added to make sure we don't accidentally change any existing behavior and use `proxyFetch` only when its explicitly set by user.

Reply:
ok, then we want to also check for HTTP_PROXY, and check for NO_PROXY for a url as well - which internally proxyFetch does (via https://www.npmjs.com/package/proxy-from-env). Would be easier to delegate this to that.